app/routes/synonyms/synonyms.server.ts

```diff
@@ -175,6 +220,8 @@ export async function putSynonyms({
     },
   }
   await client.batchWrite(params)
+  response.success = true
+  return response
```

Comment:
The function should succeed and return, or fail and never return (i.e., raise an error).